phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/formats/hpo/category/HpoCategory.java

```diff
@@ -13,60 +13,90 @@
  * gastrointestinal can have the subcategory liver.</p>
  * <p>The main use case for this class is to display a set of Terms that descend from this Category. The class
  * is intended to be used with HpoCategoryMap to display a set of terms that can belong to </p>
+ *
  * @author <a href="mailto:peter.robinson@jax.org">Peter Robinson</a>
  * @version 0.1.14
  */
+@Deprecated(since = "3.0.0", forRemoval = true)
```

Comment:
The class should be deprecated as of this version (`2.0.6`). I'll push a change where I'll update this, and I'll also add @deprecated JavaDoc tag where we will suggest the replacement. The suggestion makes it easier for the user to update the code as soon as he notices the deprecation.

Otherwise, I believe the remaining changes of this class are just formatting updates. Please let me know if I'm wrong.